Emby.Server.Implementations/Images/BaseDynamicImageProvider.cs

```diff
@@ -122,6 +122,7 @@ protected async Task<ItemUpdateType> FetchToFileInternal(
             }
 
             await ProviderManager.SaveImage(item, outputPath, mimeType, imageType, null, false, cancellationToken).ConfigureAwait(false);
+            File.Delete(outputPath);
```

Comment:
Seems a bit odd to save the file and then immediately delete it...

Reply:
> Seems a bit odd to save the file and then immediately delete it...

Because output path here is always a temp folder and the save function is what to save it into the metadata folder